### DISCUSSION TITLE
No traceable action

### DISCUSSION THREAD

**Discussion by sebastianpfischer** [community/community#36387] [Actions]
Hello,

in our project, we are really confused about a specific action we created.

The action is supposed to trigger on specific conditions: https://github.com/eclipse/kiso-testing/blob/master/.github/workflows/changelog.yml

But sometimes, it is also triggered on some PR: https://github.com/eclipse/kiso-testing/pull/185/commits (See commit called [chore(changelog): update changelog to newest version](https://github.com/eclipse/kiso-testing/pull/185/commits/d30212675abd5da5e60a6c021dd2a6c8ded3a3a4))
But when we check the Checks: https://github.com/eclipse/kiso-testing/pull/185/checks we do not see the action that triggered the commit!

We searched around the project, also in https://github.com/eclipse/kiso-testing/actions/workflows/changelog.yml and we cannot find it!

In the case of the other PR, for example https://github.com/eclipse/kiso-testing/pull/184, we do not see the commit.

Somehow we are not able to identify the conditions that trigger the action nor identifying where the action comes from...

Thanks in advance for your support.

**Comment by ossanna16** [comment]
Hey there @sebastianpfischer and thank you for asking a good question! I'm glad you found our community 🙂
